client/components/Players/DraggableQueue.vue

```diff
@@ -80,8 +80,9 @@ export default Vue.extend({
   methods: {
     ...mapGetters('playbackManager', ['getCurrentItem']),
     ...mapActions('playbackManager', ['setNewQueue', 'setCurrentIndex']),
+    ...mapState('playbackManager', ['currentItemIndex']),
     isPlaying(index: number): boolean {
-      return index === this.$store.state.playbackManager.currentItemIndex;
+      return index === (this.currentItemIndex as unknown);
```

Comment:
This should probably mirror the type defined in the store, instead.

Reply:
The problem is that mapState returns ``Computed`` as type, so it doesn't have the proper typings. I hope this is fixed in Vuex 4.

Reply:
```suggestion
      return index === (this.currentItemIndex as number | null);
```

What I mean is doing this, so we get the proper types, just in case :)

Reply:
Needs conversion to unknown first as the types don't overlap, will try later ``(this.currentItemIndex as unknown) as number``